### ISSUE TITLE
Add support for permanent and signed cookies

### ISSUE THREAD

**Issue by yukeippi** [ruby-grape/grape#715]
I want to use signed cookie in Grape.
http://api.rubyonrails.org/classes/ActionDispatch/Cookies/ChainedCookieJars.html

Can I use it?
I want how it uses.

**Comment by ash14** [comment]
Hey, progress on this seems to have stalled? I'd like to see Grape's own cookie class removed and its functionality (plus signed cookie support) moved to that gem.

Can submit a PR if need be.

**Comment by dblock** [comment]
@ash14 Yes, doesn't look like anyone worked on this since, please feel free to help! Lets not worry about 1.0, we have a normal upgrade path for backwards incompatible things, see UPGRADING.
